code/modules/mob/inventory.dm

```diff
@@ -43,7 +43,7 @@
 /mob/proc/put_in_l_hand(obj/item/W)
 	if(!put_in_hand_check(W))
 		return 0
-	if(!l_hand && has_left_hand())
+	if(!l_hand && has_left_hand() && !ABSTRACT)
```

Comment:
Absolutely not. `ABSTRACT` is a number (64) so doing `!64` will always evaluate to false, preventing you from putting *any* item in your hand. To check if an item has a flag, you need to do `W.flags & ABSTRACT`.

Either way, I think this is the wrong proc to put the check in. Switching the hand holding an item occurs in `/obj/item/attack_hand()`, so do take a look at that.